src/reanimated2/animation/util.ts

```diff
@@ -451,6 +447,7 @@ export function cancelAnimation<T>(sharedValue: SharedValue<T>): void {
 }
 
 // TODO it should work only if there was no animation before.
+// ts-prune-ignore-next
 export function withStartValue(
```

Comment:
Could check why do we have this piece of code?

Reply:
It seems to be a legacy code, which should be cleaned up 🧹🪣